sprocs/check_assessment_access.sql

```diff
@@ -38,7 +38,7 @@ BEGIN
                     END)
             ELSE 'None'
         END AS credit_date_string,
-        aar.time_limit_min,
+        LEAST(aar.time_limit_min, extract(epoch FROM aar.end_date - now()) / 60)::integer AS time_limit_min,
```

Comment:
We only want to set a time limit (that is, for `time_limit_min` to be non-NULL) if the access rule actually has a time limit in it. Because `LEAST(NULL, 3)` returns `3`, I think the code above will set a time limit equal to the time-until-endDate even when no explicit time limit was set.

Reply:
Tested it and you're right. Wrapped it in a `CASE` block. Not sure if there's a better way.